algorithms/gr-bfsrh/javascript/bfs.js

```diff
@@ -0,0 +1,42 @@
+function bfs(graph, root) {
```

Comment:
Codacy found an issue: [Function 'bfs' has a complexity of 6. Maximum allowed is 5.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6319059)